server/api/map_handler.py

```diff
@@ -0,0 +1,71 @@
+
+import json
+import smtplib
+import requests
+from flask import jsonify, request, Blueprint
+from config import API_KEY
+map_handler = Blueprint('map_handler', __name__)
+
+@map_handler.route('/map', methods=['POST'])
+def map():
+    if request.method != 'POST':
+        return 'NOT POST'
+    
+    postData = json.loads(request.get_data())
+    address = postData['address'].replace(' ', '+')
+    
+    url = "https://maps.googleapis.com/maps/api/geocode/json?address=%s&key=%s" % (address, API_KEY)
+    googleReq = requests.get(url).json()
+
+    #check for bad request
+    #check if location exists
+    googleHandler = googleAPIHandler()
+    status = googleHandler.handle(googleReq)
+    if status != True:
+        print(status['log'])
+        return status['msg']
+    
+    #get geocodes
+    geocodes = googleReq['results'][0]['geometry']['location']
+    #get bounds: viewport -> latitude,longitude values defining the southwest and northeast corner of the viewport bounding box
+    viewport = googleReq['results'][0]['geometry']['viewport']
+    
+    #return geolocation
+    return {'geocodes':geocodes,'viewport':viewport}
```

Comment:
@SilverStrings024 is using `make_response` to generate the responses. You guys should discuss and decide on one way to handle returning responses so the code is more uniform. Does this even work properly though? Generally you have to do some kind of `jsonify(response)` or something.

Reply:
I've changed it